README.md

```diff
@@ -1 +1,174 @@
-# bedrock-stats-monitor-os
\ No newline at end of file
+# bedrock-stats-monitor-os
+
+## Configuration
+Metrics may be any of the APIs in the
+[systeminformation](https://www.npmjs.com/package/systeminformation#reference)
+module.
+```
+bedrock.config['stats-monitor-os'].metrics = {
+  currentLoad: true,
+  fsSize: true,
+  mem: true,
+};
+```
+
+## Sample Report with Default Metrics
+```
```

Comment:
This is verbose for a README.  Perhaps at least simplify down to one CPU?

Reply:
moved to docs folder.